### PR TITLE
Proper submission count on CSV imports

### LINKED ISSUE #1065: CSV Imports, update submission count correctly after successful import
- [x] update `XFrom.num_of_submissions` when csv import completes
- [x] update accordingly after every X(lets start with 100/50?) records.
- [x] if an exception happens, send email to admins for further analysis

### PR DESCRIPTION
**Pull request by ukanga**
closes #1065

### CODE REVIEW

File: onadata/libs/utils/logger_tools.py [hunk 1]
```diff
@@ -1,59 +1,48 @@
...
 from onadata.apps.logger.xform_instance_parser import (
-    InstanceEmptyError,
-    InstanceInvalidUserError,
-    InstanceMultipleNodeError,
-    NonUniqueFormIdError,
-    DuplicateInstance,
-    clean_and_parse_xml,
-    get_uuid_from_xml,
-    get_deprecated_uuid_from_xml,
-    get_submission_date_from_xml)
```
Comment (pld) [inline onadata/libs/utils/logger_tools.py]: I actually prefer this style, but either consistently is fine, would be nice to define this in the style checker

Comment (ukanga) [inline onadata/libs/utils/logger_tools.py]: This is the work of [yapf](https://github.com/google/yapf), I will add a config for it.

File: onadata/libs/utils/csv_import.py [hunk 2]
```diff
@@ -266,29 +269,44 @@ def submit_csv(username, xform, csv_file):
...
-                        meta={
-                            'progress': additions,
-                            'total': num_rows,
-                            'info': addition_col
-                        })
-                except:
-                    pass
+                if additions % getattr(settings,
+                                       'EXPORT_TASK_PROGRESS_UPDATE_BATCH',
+                                       100) == 0:
```
Comment (ukanga) [inline onadata/libs/utils/csv_import.py]: Noted, will set a static variable for this.

File: onadata/libs/utils/logger_tools.py (hunk 1 above)
Comment (ukanga) [inline onadata/libs/utils/logger_tools.py]: Using the default pep8 style I think.

**Pull request merged**